frontend/src/components/table/TableCell.js

```diff
@@ -431,6 +359,7 @@ TableCell.propTypes = {
   viewId: PropTypes.string,
   modalVisible: PropTypes.bool,
   docId: PropTypes.any,
+  activeLocale: PropTypes.object,
```

Comment:
This is because activeLocale is passed from the parent component. Also note that this was branched from a branch that did not had the changes added in utils/locale.js. This will be fixed along with the merging on which @siemiatj is currently working on.

Reply:
@petrican @siemiatj  ok, but just to be sure.

At the end i would expect to have the locale 
* only in local store and accessed via `getCurrentActiveLocale` function
* or we have it in redux state and accessed from there or mapped to properties.

What I would really avoid is having it in both places.

PS: personally I think the best way would be to have it in redux store...

Reply:
Ok I will sync with @siemiatj after everything is merged together to make sure there's only one way of getting the locale across the app. Thanks